src/piadm.sh

```diff
@@ -80,8 +81,7 @@ declare installstamp
 
 poolpresent() {
 	# Works for an empty $1, which is "all of them" or "unspecified"
-	zpool list $1 > /dev/null 2>&1
-	if [[ $? -ne 0 ]]; then
+	if ! zpool list "$1" > /dev/null 2>&1 ; then
```

Comment:
Would losing the quotes here be a problem?  I removed them from my local copy and my prior problems disappeared.

Reply:
Either that or you'll need to special-case an empty $1.

Reply:
special casing this for safety